src/molecules/Table/Table.test.tsx

```diff
@@ -0,0 +1,124 @@
+import { mount, ReactWrapper, ShallowWrapper } from 'enzyme';
+import React from 'react';
+import renderer from 'react-test-renderer';
+import 'jest-styled-components';
+
+import { Table } from './Table';
+import { Badge } from 'atoms';
+
+describe('Table', () => {
+  let wrapper: ReactWrapper | ShallowWrapper;
+
+  const args = {
+    columns: [
+      {
+        Header: 'Column 1',
+        accessor: 'col1' // accessor is the "key" in the data
+      },
+      {
+        Header: 'Column 2',
+        accessor: 'col2'
+      },
+      {
+        Header: 'Column 3',
+        accessor: 'col3'
+      }
+    ],
+    data: [
+      {
+        col1: <img src="https://picsum.photos/75" />,
+        col2: 'Row 1 Cell 2',
+        col3: 'Row 1 Cell 3'
+      },
+      {
+        col1: 'Row 2 Cell 1',
+        col2: <a href="#">Link</a>,
+        col3: 'Row 2 Cell 3'
+      },
+      {
+        col1: 'Row 3 Cell 1',
+        col2: 'Row 3 Cell 2',
+        col3: <Badge status="neutral">Badge</Badge>
+      }
+    ]
+  };
+
+  beforeEach(() => {
+    wrapper = mount(<Table columns={args.columns} data={args.data} />);
+  });
+
+  describe('basic table structure', () => {
+    it('renders the table', () => {
+      expect(wrapper.find('table[role="table"]').exists()).toBeTruthy();
+      expect(wrapper.find('thead').exists()).toBeTruthy();
+      expect(wrapper.find('tbody[role="rowgroup"]').exists()).toBeTruthy();
+      expect(
+        wrapper.find('table thead tr th PureIcon[icon="unfold"]')
+      ).toHaveLength(args.columns.length);
+    });
+  });
+
+  describe('unsorted table data', () => {
+    // column headers
+    it('renders the header content', () => {
+      const len = args.columns.length;
+      const columns = Array.from(Array(len), (x, index) => index);
+
+      columns.forEach(column => {
+        const text = args.columns[column].Header;
+        expect(wrapper.find('table thead tr th').at(column)).toHaveText(text);
+      });
+    });
```

Comment:
This can be written a lot simpler.

```suggestion
    it('renders the header content', () => {
      args.columns.forEach((column, i) => {
        expect(wrapper.find('th').at(i)).toHaveText(column.Header);
      });
    });
```

Reply:
Also this test doesn't need to be in a 'unsorted table data' describe block, the header column text has nothing to do with the sort order, right?

Reply:
I had it that way because I was originally matching the data in the test file which would require additional logic to account for the changing chevron icons, but since I'm doing a snapshot now, you're right that it's no longer relevant. I will update.